### ISSUE TITLE
Top Gear 3000 graphic glitches

### ISSUE THREAD

**Issue by N0ciam** [libretro/snes9x2010#122]
I wonder if anyone could make this game work? Or if any programmer has the knowledge to get the solution from the newer Snes9x and put in that version.

**Comment by hizzlekizzle** [comment]
It's not really something that can be backported easily. Unfortunately, some bugs/glitches like this are the price we pay for snes9x2010's speed as compared with the more accurate mainline version.

**Comment by N0ciam** [comment]
I was looking for a functional emulator. I found, for those looking for an emulator that ran this game just download the source and compile, the last binary is still in error. lr-snes9x
https://github.com/libretro/snes9x

For programmers, if you want to study the solution adopted by the other emulator here is the link. 
https://github.com/libretro/snes9x/pull/169/commits/cb04b1ff4fa2196c51015699af9e4a4bcc4ca6db

I believe the topic has been closed without any analysis, but thank you anyway.
